.github/workflows/create_release.yaml

```diff
@@ -11,6 +11,8 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

Comment:
is this needed? I'm pretty sure we can use x86 to build the arm binaries just fine.

Reply:
We can cross build the arm binaries with Go easily, but this is building the binaries in docker using the existing build system which does require qemu.

Reply:
alrighty